cms/jinja2/templates/base_iframe.html

```diff
@@ -0,0 +1,9 @@
+{% extends "layout/_template.njk" %}
```

Comment:
I could not find anything using this template. One we missed in #229?

Reply:
I don't know what the plans for future iframe visualisation support are, but I don't believe they will use this. This is removed in aa70fd2662ccae54e66d3fdc14986904b3858a61.